_posts/2019-01-23-multifile-python-functions.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Multifile python functions
+description: "Tidy python in an OpenFaaS world; or, what happens when you want to split your Python function into multiple modules?"
+date: 2019-01-23
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
+categories:
+  - python
```

Comment:
What other categories apply?

Reply:
i am not sure, it seems pretty specific to just python.  everything should work with any provider. Perhaps, `best-practice` and `development`?

Reply:
* `serverless`
* `coding`
* `examples`